worker/src/durable.rs

```diff
@@ -737,12 +768,40 @@ impl DurableObject for Chatroom {
 }
 ```
 */
+
 #[async_trait(?Send)]
 pub trait DurableObject {
     fn new(state: State, env: Env) -> Self;
+
     async fn fetch(&mut self, req: Request) -> Result<Response>;
+
     #[allow(clippy::diverging_sub_expression)]
     async fn alarm(&mut self) -> Result<Response> {
         unimplemented!("alarm() handler not implemented")
     }
+
+    #[allow(unused_variables, clippy::diverging_sub_expression)]
```

Comment:
I think you can name the variables with `_ws` in the trait to silence `clippy`, and then implementors of the trait can still use whatever name they want.

Reply:
On the other hand, the variable name will appear in the documentation, so it's a tradeoff-- is it better to have a clean variable name in the docs or to avoid unnecessary lint `allow`s?

Reply:
As Eric said, that will cause the argument to appear as `_ws` both in documentation and in inline argument name hints.